src/soap.js

```diff
@@ -554,7 +554,8 @@ class SoapMethodCall {
             url: url,
             method: 'POST',
             headers: headers,
-            data: DomUtil.toXMLString(this._doc)
+            data: DomUtil.toXMLString(this._doc),
+            ...(this._pushDownOptions && {signal : this._pushDownOptions.signal})
```

Comment:
I do not think this is necessary. The pushdown mechanism should pass the signal attirbute automatically down to the transport layer